geode-lucene/src/test/java/org/apache/geode/cache/lucene/internal/cli/LuceneIndexCommandsDUnitTest.java

```diff
@@ -706,16 +683,17 @@ private String executeCommandAndLogResult(final CommandStringBuilder csb) {
   private CommandResult executeCommandAndGetResult(final CommandStringBuilder csb) {
     String commandString = csb.toString();
     writeToLog("Command String :\n ", commandString);
-    CommandResult commandResult = executeCommand(commandString);
-    String resultAsString = commandResultToString(commandResult);
+    CommandResult commandResult = gfsh.executeAndVerifyCommand(commandString);
+    String resultAsString = gfsh.getGfshOutput();
     writeToLog("Result String :\n ", resultAsString);
```

Comment:
if both writeToLog are not necessary (gfsh.executexxx() already has logs), this method can be replaced with just gfsh.executeAndVerify()